app/jobs/import_url_job.rb

```diff
@@ -31,10 +31,9 @@ def perform(file_set, operation)
       # on a machine that did not have this temp file on it's file system.
       # NOTE: The return status may be successful even if the content never attaches.
       if Hyrax::Actors::FileSetActor.new(file_set, user).create_content(f, 'original_file', false)
-        # send message to user on download success
-        Hyrax.config.callback.run(:after_import_url_success, file_set, user)
```

Comment:
Don't bother sending a notification if the import succeeded. User will already get notified when the FileSet is attached to the Work.